src/status_im/utils/slurp.clj

```diff
@@ -10,6 +11,29 @@
        (map (fn [file-name]
               (try
                 (clojure.core/slurp
-                  (s/join "/" ["resources/js/bots" (name bot-name) file-name]))
+                  (string/join "/" ["resources/js/bots" (name bot-name) file-name]))
                 (catch Exception _ ""))))
        (apply str)))
+
+(def svg-tags #{:g :rect :path :use :defs})
+
+(defmacro slurp-svg [file]
```

Comment:
What is the purpose of this macro? Some documentation and examples would be useful.

Also, can we pull out util functions and make the macro part smaller?

Reply:
ok

Reply:
I agree with @oskarth, this needs some docstring and explanation.

Reply:
done